pkg/rootless/rootless_linux.go

```diff
@@ -172,3 +188,86 @@ func BecomeRootInUserNS() (bool, int, error) {
 
 	return true, int(ret), nil
 }
+
+func readUserNs(path string) (string, error) {
+	b := make([]byte, 256)
```

Comment:
Are we guaranteed this will be big enough?

Reply:
that is much bigger than needed, the link is something like "user:[4026531837]".  Should I make the buffer smaller?

Reply:
Oh, I thought we were potentially holding a full path. 256 is probably still fine, we aren't exactly optimizing for memory usage yet.